sources/playground/examples/repl.sql

```diff
@@ -0,0 +1,10 @@
+DECLARE PROC printf NO CHECK;
+
+CREATE PROC entrypoint ()
+BEGIN
+    declare x text not null;
+
+    SET x := 'hello';
```

Comment:
@ricomariani Is there a way to always require a database even if there is no need for it? I wanted the repl.sql to be empty but the signature of the procedure changes.

```sql
DECLARE PROC printf NO CHECK;

CREATE PROC entrypoint ()
BEGIN
    call printf("hello world"); -- breaks
END;
```

```sql
DECLARE PROC printf NO CHECK;

CREATE PROC entrypoint ()
BEGIN
    call printf((SELECT "hello world" as x)); -- works
END;
```

Is there something like this that I could use?
`CREATE PROC entrypoint () USING TRANSACTION`, `@attribute(cql:using_transaction)` or `@attribute(cql:use_database)`

Reply:
I'll do this:

```
cat ./out/repl/repl.h | grep "entrypoint(void)" >/dev/null 2>&1 && echo "void" || echo "uses db"
```